### PR TITLE
Able to register routes or not in the config

### PR DESCRIPTION
**Pull request by vincent-tarrit**
Hello, I'm using filament so the route is not necessary in my case. I added a config params to remove this routes:) 

Thank you very much for your job ! It's great !

### CODE REVIEW

**Review by andreaselia — CHANGES_REQUESTED** [review]
Thanks @vincent-tarrit, I'm happy to add the ability to disable the route, but would need the config parameter adding to the config, as well as a test case if possible. StyleCI is also failing at the moment.

**Comment by vincent-tarrit** [comment]
yes, sorry just added it :)

File: config/analytics.php [hunk 1]
```diff
@@ -41,4 +41,12 @@
...
     ],
 
+    'registerRoutes' => true,
+
+    'translations' => [
+        'devices' => 'Devices',
+        'pages' => 'Pages',
+        'sources' => 'Sources',
+        'users' => 'Users',
+    ],
```
Comment (andreaselia) [inline config/analytics.php:51]: This isn't part of this PR.

File: src/AnalyticsServiceProvider.php [hunk 2]
```diff
@@ -33,10 +33,12 @@ public function boot(): void
             Analytics::class,
         ]);
 
-        // Routes
-        Route::group($this->routeConfig(), function () {
-            $this->loadRoutesFrom(__DIR__.'/../routes/web.php');
-        });
+        if (config('analytics.registerRoutes')){
```
Comment (andreaselia) [inline src/AnalyticsServiceProvider.php:36]: Could you add a PHPUnit feature test for this?

**Review by andreaselia — CHANGES_REQUESTED** [review]

**Comment by andreaselia** [comment]
Hi mate, I'm closing this for the time being, feel free to re-open it in the future with the necessary changes. Thanks!

**Pull request closed without merging**